torch/csrc/jit/serialization/import_export_constants.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <string>
```

Comment:
I don't think this is the right place for this :) but also not sure what the purpose is...

Reply:
The major purpose for BYTECODE_INDEX_... constants, is that they are used both in exporting module and exporting op names. As you suggested in my previous PR, I replaced the inline numbers with those constants.

Reply:
Right, but I'd move this include out to the source files that use it, rather than including it here - it's not needed for the definitions in this file.